x-pack/platform/plugins/shared/fleet/server/types/models/agent_policy.ts

```diff
@@ -158,6 +158,18 @@ export const AgentPolicyBaseSchema = {
       ),
     })
   ),
+  fleet_ui: schema.maybe(
+    schema.object({
+      required_versions: schema.maybe(
```

Comment:
nit: it could be nice to add a description to those fields explaining it's related to upgrade the feature